src/components/CreateProfile/WorkInformation/WorkInformation.tsx

```diff
@@ -77,23 +73,23 @@ const WorkInformation = () => {
           Employment type
           <CheckboxInput
             id="fulltime"
-            label="Full-time"
-            checked={values.employment.includes('Full-time')}
-            onChange={() => handleEmploymentType('Full-time')}
+            label="FULL_TIME"
```

Comment:
**nitpick:** This should be a `const` or enum to prevent duplicating this string